File: plugin/BanCount/src/main/java/BanCount.java
import android.content.Context;
import com.aliucord.Utils;
import com.aliucord.annotations.AliucordPlugin;
import com.aliucord.entities.Plugin;
import com.discord.widgets.servers.WidgetServerSettingsBans;

@SuppressWarnings("MISSING_DEPENDENCY_SUPERCLASS")
@AliucordPlugin
public class BanCount extends Plugin {
    @Override
    public void start(Context context) {
        patcher.after(WidgetServerSettingsBans.class, "configureUI", (callFrame) -> {
            WidgetServerSettingsBans widget = (WidgetServerSettingsBans) callFrame.thisObject;
            WidgetServerSettingsBans.Model model = (WidgetServerSettingsBans.Model) callFrame.args[0];
            
            widget.actionBarTitleLayout.setSubtitle(
                model.getGuildName() + " (" +
                Utils.pluralise(model.getTotalBannedUsers(), "ban") +
                ")"
            );
        }, WidgetServerSettingsBans.Model.class);
    }

    @Override
    public void stop(Context context) {
        patcher.unpatchAll();
    }
}

File: plugin/BanCount/src/main/kotlin/BanCount.kt
import android.content.Context
import com.aliucord.Utils
import com.aliucord.annotations.AliucordPlugin
import com.aliucord.entities.Plugin
import com.aliucord.patcher.after
import com.aliucord.patcher.component1
import com.aliucord.patcher.component2
import com.discord.widgets.servers.WidgetServerSettingsBans

@AliucordPlugin
class BanCount : Plugin() {
    override fun start(context: Context) {
        patcher.after<WidgetServerSettingsBans>(
            "configureUI",
            WidgetServerSettingsBans.Model::class.java
        ) { (_, model: WidgetServerSettingsBans.Model) ->
            actionBarTitleLayout.setSubtitle(
                "${model.guildName} (${
                    Utils.pluralise(
                        model.totalBannedUsers,
                        "ban"
                    )
                })"
            )
        }
    }

    override fun stop(context: Context) = patcher.unpatchAll()
}